docker/libra-build.sh

```diff
@@ -4,17 +4,20 @@
 set -e
 
 DOCKERFILE=$1
+if [ -n "ENABLE_FAILPOINTS" ]; then
+  TAG_SUFFIX="_failpoints"
+fi
 if [ -z "$LIBRA_BUILD_TAG" ]; then
-  TAGS="--tag $2"
+  TAGS="--tag $2$TAG_SUFFIX"
 else
-  TAGS="--tag $2 --tag $LIBRA_BUILD_TAG"
+  TAGS="--tag $2$TAG_SUFFIX --tag $LIBRA_BUILD_TAG$TAG_SUFFIX"
 fi
 
 RESTORE='\001\033[0m\002'
 BLUE='\001\033[01;34m\002'
 DOCKERFILE_BUILD_RE='^RUN ./docker/build-common.sh$'
 
-if [ -z "$DOCKERFILE" ] || [ -z "$TAG" ]
+if [ -z "$DOCKERFILE" ] || [ -z "$TAGS" ]
```

Comment:
how is this currently passing? TAG must be defined somewhere, no? and then what the hell is $TAGS doing? Would recommend applying the suffix on old line 12 without adding an additional variable...


```
if [ -n "$ENABLE_FAILPOINTS" ]; then
  TAGS="$TAGS""_failpoints"
fi